src/OrchardCore.Modules/OrchardCore.Menu/Models/ContentPickerMenuItemPart.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.AspNetCore.Routing;
+using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement.Routing;
+
+namespace OrchardCore.Menu.Models
+{
+    public class ContentPickerMenuItemPart : ContentPart
+    {
+        /// <summary>
+        /// The name of the link
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Content Picker ItemIds
+        /// </summary>
+        public string ContentItemIds { get; set; }
+
+        public RouteValueDictionary GetContentPickerItemRouteValue(AutorouteOptions autoRouteOption)
+        {
+            var routeValues = new RouteValueDictionary(autoRouteOption.GlobalRouteValues);
```

Comment:
This method should be somewhere else, `ContentPickerMenuItemPart` is serializable, so shouldn't have methods on it